todo_app/app.py

```diff
@@ -8,17 +9,44 @@
 from todo_app.flask_config import Config
 from flask import request
 import requests
-from todo_app.view_model import ViewModel    
+from todo_app.view_model import ViewModel 
+from flask_login import current_user, login_user, logout_user, LoginManager, login_required, UserMixin
+from oauthlib.oauth2 import WebApplicationClient  
```

Comment:
A few of these imports (including this WebApplicationClient) look to be unused, so it would be nice to clear those out when you get a chance

Reply:
Done